controller_manager/src/controller_manager.cpp

```diff
@@ -811,9 +910,12 @@ std::vector<std::string> ControllerManager::get_controller_names()
   return names;
 }
 
-controller_interface::return_type
-ControllerManager::update()
+controller_interface::return_type ControllerManager::update(bool update_resources)
```

Comment:
Why did you add this argument?

Reply:
well, that way a simple timer callback will take care of a complete cycle: `read, update, write`. However, some people might fancy more control over this and treat the controller manager solely responsible for updating the controllers. So the read/write can be done externally.

Reply:
I'd prefer if we didn't include this here, better keep things clean. We have the example node to take care of this.